perllib/FixMyStreet/App.pm

```diff
@@ -318,15 +319,15 @@ sub send_email {
         ]
     };
 
+    return if $c->is_abuser($vars->{to});
+
     # render the template
     my $content = $c->view('Email')->render( $c, $template, $vars );
 
     # create an email - will parse headers out of content
     my $email = Email::Simple->new($content);
-    $email->header_set( ucfirst($_), $vars->{$_} )
-      for grep { $vars->{$_} } qw( to from subject Reply-To);
-
-    return if $c->is_abuser( $email->header('To') );
+    $email->header_set( 'Subject', $vars->{subject} ) if $vars->{subject};
+    $email->header_set( 'Reply-To', $vars->{'Reply-To'} ) if $vars->{'Reply-To'};
```

Comment:
I think this should be okay, as Reply-To only ever contains one item, and Email::Simple will take the first given an arrayref. For consistency, might want to move down to From/To conditionally, but shrug.